mne/channels/channels.py

```diff
@@ -298,9 +309,12 @@ def set_eeg_reference(self, ref_channels=None, verbose=None):
         Returns
         -------
         inst : instance of Raw | Epochs | Evoked
-            Data with EEG channels re-referenced. For ``ref_channels=None``,
-            an average projector will be added instead of directly subtarcting
-            data.
+            Data with EEG channels re-referenced. If ``ref_channels='average'``
+            and ``projection=True`` a projection will be added instead of
+            directly re-referencing the data.
+        ref_data : array
```

Comment:
Looks like this is not returned, since you pick the first element in ``return``.